QubitServer/src/main/java/org/labrad/qubits/channeldata/IqDataBase.scala

```diff
@@ -0,0 +1,33 @@
+package org.labrad.qubits.channeldata
+
+import org.labrad.qubits.channels.IqChannel
+
+abstract class IqDataBase extends IqData {
+
+  private var channel: IqChannel = _
```

Comment:
in a variable initialization expression like this, underscore means the default 'zero' value for the type in question. Here, that would be `null`. I should probably just be explicit here and write null, there's really nothing to gain from using underscore.